test-utils/src/main/java/org/sonarsource/sonarlint/core/test/utils/server/ServerFixture.java

```diff
@@ -19,6 +19,23 @@
  */
 package org.sonarsource.sonarlint.core.test.utils.server;
 
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
```

Comment:
I think your formatting settings might be broken, static imports normally come after the normal ones